examples/two-columns-layout/src/components/Icons.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx h */
+import { h } from 'preact';
+
+type IconProps = {
+  className?: string;
+};
+
+export const ChevronRightIcon = ({ className }: IconProps) => (
```

Comment:
We use [function declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function) in the codebase instead of expressions.